authentication-ldap/src/main/java/com/synopsys/integration/alert/authentication/ldap/validator/LDAPConfigurationValidator.java

```diff
@@ -35,4 +36,29 @@ public ValidationResponseModel validate(LDAPConfigModel ldapConfigModel) {
 
         return ValidationResponseModel.success();
     }
+
+    public ValidationResponseModel validate(LDAPConfigTestModel ldapConfigTestModel) {
+        Set<AlertFieldStatus> statuses = new HashSet<>();
+        LDAPConfigModel ldapConfigModel = ldapConfigTestModel.getLdapConfigModel();
+
+        if (null == ldapConfigModel) {
+            statuses.add(AlertFieldStatus.error("ldapConfigModel", AlertFieldStatusMessages.REQUIRED_FIELD_MISSING));
```

Comment:
Does an "ldapConfigModel" key exist in the UI? I see that we have fields for "testLDAPUsername" and and "testLDAPPassworrd" in AuthenticationModel.js but nothing for the model itself.

Reply:
When the test button is used, the UI passes in an LDAPConfigTestModel, which contains an LDAPConfigModel.

Reply:
When we create an AlertFieldStatus we're associating some error/warning message to some field in the UI. In this case the field is "ldapConfigModel". Typically its a single unit of information thats passed in a text or number field (think username/password/port/etc.).  When an error occurs we can add an AlertFieldStatus to show a red error message. This one is a bit more complex since it refers to the entire model.

In this case, I'm unable to find a field in the UI with the tag "ldapConfigModel" so I'm not sure if we post any kind of field error using it.